BrianLusina/kvault
# Fix protocol handler

## Description

Fixes protocol handler and removes unused classes

## How can we test this?

1. Run `kvault` server with `python kvault.py` with optional provided arguments as specified in the README
2. In a separate terminal session, use the client to make connections to the server and perform operations, also as described in the README.
3. For every value saved, it should be possible to retrieve it from the client.

## Checklist:
